internal/commands/result.go

```diff
@@ -482,13 +488,25 @@ func CreateScanReport(
 	return nil
 }
 
+func isScanPending(scanStatus string) bool {
+	scanPendingStatus := true
+	if strings.EqualFold(scanStatus, "Completed") || strings.EqualFold(scanStatus, "Partial") || strings.EqualFold(scanStatus, "Failed") {
+		scanPendingStatus = false
+	}
+	return scanPendingStatus
```

Comment:
Can be simplified to
`return !(strings.EqualFold(scanStatus, "Completed") || strings.EqualFold(scanStatus, "Partial") || strings.EqualFold(scanStatus, "Failed"))`